Carbonite.Quests/Data/Quests1-10.lua

```diff
@@ -1,4 +1,4 @@
-			
```

Comment:
Please add only changes for QuestItems update. Those quest changes are not needed.